app/models/partner.rb

```diff
@@ -203,6 +215,21 @@ def contact_person
     }
   end
 
+  def agency_info
+    return @agency_info if @agency_info
+
+    return {} if profile.blank?
+
+    @agency_info = {
+      address: [profile.address1, profile.address2].reject(&:blank?).join(', '),
```

Comment:
I usually prefer positive over negative statements, so I'd like this to be `.select(&:present?)`

Reply:
it's definitely make the code more readable. Thanks